notebooks/Revised_Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -0,0 +1,1528 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[      fun: 45894.07068100051\n",
+      " hess_inv: <1x1 LbfgsInvHessProduct with dtype=float64>\n",
+      "      jac: array([0.00239975])\n",
+      "  message: 'CONVERGENCE: REL_REDUCTION_OF_F_<=_FACTR*EPSMCH'\n",
+      "     nfev: 8\n",
+      "      nit: 5\n",
+      "     njev: 8\n",
+      "   status: 0\n",
+      "  success: True\n",
+      "        x: array([0.006875])]\n",
+      "[45894.07068100051]\n",
+      "[[0.006875003655224625]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "#%%time\n",
+    "pars, nllh, res = sig_miner.minimize()\n",
+    "\n",
+    "print(res)\n",
+    "print(nllh)\n",
+    "print(pars)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The log likelihood is calculated for the background-only model so that it can be compared to the compound model. Setting the parameter 'Signal_A' to 1e-10 effectively cancels out the contribution from the signal model. Since we are not varying any of the parameters, there the log likelihood only needs to be calculated once using the .get_logprob() function.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 102,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "46038.74332720728\n",
+      "17.01015262757948\n",
+      "CPU times: user 3.43 ms, sys: 1.41 ms, total: 4.84 ms\n",
+      "Wall time: 4.07 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# Evaluating log likelihood for background only model. This is accomplished by setting Signal_A to effectively zero. \n",
+    "pars_['Signal_A'] = 1e-10\n",
+    "bkg_nllh = -sig_llh_obj.get_logprob(pars_)\n",
+    "print (bkg_nllh)\n",
+    "print (sqrtTS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The likelihood ratio test statistic given by:\n",
+    "\n",
+    "$$\\lambda  = -2[LLH(\\Theta_{bkg}) - LLH(\\Theta_{sig}, \\Theta_{bkg})]$$\n",
+    " \n",
+    " (citate the paper)\n",
+    "\n",
+    "is used to compare the compare the background+signal model to the background model. This statistic is used to quantify the signicance of the signal+background model. In the following block, the likelihood ratio test statistic is calculated using the negative log likelihood found for the background model and the minimized background+signal model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 103,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "17.01015262757948\n"
```

Comment:
This is a confident detection, a strong preference for there being a (GRB) signal in the data.
Compare it to Figure 19 in the paper and you will see that the rate of a false signal with this sqrt(TS) is much much below 1 per day of data searched (off the end of the plot). Our typical threshold for confident detection of a GRB is sqrt(TS) of 8.
provide the reader some context on this number!